CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Braintree Android SDK Release Notes
 
+## unreleased
+
+* Add `setCountryCode` to `GooglePayRequest`
+* Add Google Pay support for Elo cards. 
```

Comment:
```suggestion
* Google Pay
  * Add `setCountryCode` to `GooglePayRequest`
  * Add Google Pay support for Elo cards. 
```

I've noticed that in the web changelogs we usually group entries by feature. It might be nice to do that here as well.

Reply:
We discussed this and decided for now organizing by change type make more sense, but we'll do a CHANGELOG pass before the next beta release and potentially update all the entries to be organized by module.